config/site_settings.yml

```diff
@@ -1848,6 +1848,9 @@ developer:
   enable_safe_mode:
     default: true
     client: true
+  suppress_anonymous_pageviews_on_private_sites:
```

Comment:
Can we name the setting simply `suppress_anonymous_pageviews`? Actually, `suppress` sounds like the pageviews are deleted, maybe `ignore_anonymous_pageviews` is clearer?

Reply:
> maybe `ignore_anonymous_pageviews` is clearer?

👍